types/styled-theming/styled-theming-tests.ts

```diff
@@ -0,0 +1,13 @@
+import theme from "styled-theming";
```

Comment:
Since this is a callable or constructable entity, it isn't legal in ES2015 to import this with a namespace import like `import * as foo from 'foo'`, so could you please convert this to either

```ts
import foo = require('foo')
```

or the following with `esModuleInterop` turned on

```ts
import foo from 'foo';
```

Reply:
Thanks Daniel, I decided to go with using `"esModuleInterop": true`.